arch/arm/src/Makefile

```diff
@@ -93,8 +93,8 @@ LDFLAGS += $(ARCHSCRIPT) $(EXTRALINKCMDS)
 
 # Override in Make.defs if linker is not 'ld'
```

Comment:
Now misleading comment

Reply:
Actually this comment still applies, since the GCC driver will call LD to perform the linking process.